_dashboards/discover/multi-data-sources.md

```diff
@@ -41,22 +41,45 @@ To create a new data source connection:
 
 1. Open Dashboards. If you’re not running the security plugin, go to [`http://localhost:5601`](http://localhost:5601/). If you’re running the security plugin, go to [`https://localhost:5601`](https://localhost:5601/) and log in with the username `admin` and password `admin`.
 
-1. In the Dashboards console, choose **Stack Management** > **Data Sources `Experimental`** > **Data Sources** > **Create data source connection**, as shown in the following image.
+1. In the Dashboards console, navigate to **Stack Management** > **Data Sources `Experimental`** > **Data Sources**, and then select **Create data source connection**, as shown in the following image.
 
     <img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-1.png" width=500 alt="Data sources user interface">
 
-1. Add information to each field to configure **Connection Details**, **Endpoint**, and **Authentication** to connect to a data source, as shown in the following image. For this tutorial, the **Endpoint URL** is `http://localhost:5601/app/management/opensearch-dashboards/dataSources`.
+1. Add information to each field to configure **Connection Details**, **Endpoint URL**, and **Authentication Method**, as shown in the following image. For this tutorial, the **Endpoint URL** is `http://localhost:5601/app/management/opensearch-dashboards/dataSources`.
 
     <img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-2.png" width=500 alt="Create a data source connection user interface">
 
-1. Choose **Create data source connection** to save your settings.
-1. Return to the **Data Sources** main page to confirm that the newly created data source is listed under **Data Sources**, as shown in the following image.
+    For **Connection Details**, enter a title for the connection and the endpoint URL used to connect to the data source. A description of the connection is optional. 
+    
+    For **Authentication Method**, first select the type of authentication:
+    * **No authentication**—No authentication is used to connect to the data source.
```

Comment:
```suggestion
    * **No authentication** —No authentication is used to connect to the data source.
```

Reply:
I know it looks funky here. This is actually an em dash, and it's a little less funky on the website (although still funky). Per style guide:
"An en dash ... do not include spacing on either side. Use an en dash to ... separate a bullet heading from the following text in a list..."

Reply:
It's true that we don't use space on either side of an em dash, but we don't generally use them to separate a list item from the following text. I've changed to use colons.